addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1281,11 +1302,27 @@ const InputUserValueWidget = UnitUserValueWidget.extend({
                 }
                 value += (ev.which === $.ui.keyCode.UP ? step : -step);
                 input.value = this._floatToStr(value);
+                // We need to know if the change event will be triggered or not.
+                // Change is triggered if there has been a "natural" input event
+                // from the user. Since we are triggering a "fake" input event,
+                // it should not change the changeEventWillBeTriggered variable.
+                const changeEventWillBeTriggered = this.changeEventWillBeTriggered;
                 $(input).trigger('input');
+                this.changeEventWillBeTriggered = changeEventWillBeTriggered;
+                this.notifyValueChangeOnBlur = true;
                 break;
             }
         }
     },
+    /**
+     * @override
+     * @private
```

Comment:
I don't think `@private` is necessary on an override.

Reply:
I ve seen a lot of case where there is both but even more case where there is only the override... IDK what is the rule about that.. I'll remove it :+1:

Reply:
A private method can't become public in an override, so just override is needed.